### DISCUSSION TITLE
Problem in scrolling map with external keyboard

### DISCUSSION THREAD

**Discussion by twasilew** [osmandapp/OsmAnd#17993] [Q&A] [answered]
I use external keyboard (arrows) to scroll map.

To reproduce the issue I followed the steps
1. Connected wireless keyboard
2. Change the orientation of the map to follow my position. 
3. Then use any arrow on keyboard. 

Resoult: Map scrols random directions.

But if i change map orientation to north on top it works ok.
It must be direction of strolling in relation to map orientation.

Any one has similar problem? 

I noticed that after update on around July. 
I use version 4.5.10
On version 4.0.4 it worked fine.
On version 4.5.8 arrows keys didn't worked at all.

**Comment by dmpr0** [comment, marked as answer]
Hi!
Please check what is set in Menu – Settings – Profile – General settings – External input device

**Reply by twasilew** [reply]
Hi!
i've checked it and it didn't helped. 
It was noticed by 3 of my friends as they also use wireless keyboards.

**Comment by WebboXR** [comment]
Here I have the same issue, all ok since you press the "center map" on the keyboard or the screen, pan goes to crazy

**Reply by dmpr0** [reply]
Hi! Please try to install beta version, it should be fixed https://play.google.com/apps/testing/net.osmand, https://play.google.com/apps/testing/net.osmand.plus

**Reply by WebboXR** [reply]
I confirm that the actual beta version solve the problem
